disttask/framework/handle/handle.go

```diff
@@ -42,6 +44,9 @@ func SubmitGlobalTask(taskKey, taskType string, concurrency int, taskMeta []byte
 
 	if globalTask == nil {
 		taskID, err := globalTaskManager.AddNewGlobalTask(taskKey, taskType, concurrency, taskMeta)
+		metrics.DistDDLTaskGauge.WithLabelValues(taskType, metrics.WaitingStatus).Inc()
+		metrics.DistDDLTaskStarttimeGauge.WithLabelValues(taskType, metrics.WaitingStatus, fmt.Sprint(taskID)).Set(float64(time.Now().UnixMicro()))
+
```

Comment:
ditto.
You can add a metric pkg with metric.go which have a bunch of util functions under framwork pkg